litmus-portal/frontend/src/pages/AnalyticsPage/index.tsx

```diff
@@ -152,16 +152,17 @@ const AnalyticsPage: React.FC = () => {
                     }
                     if (chaosData.experimentVerdict === 'Fail') {
                       experimentTestResultsArray.push(0);
-                      experimentTestResultsArrayPerWorkflowRun.push(0);
                     }
                     if (
                       chaosData.experimentVerdict === 'Pass' ||
                       chaosData.experimentVerdict === 'Fail'
                     ) {
                       experimentTestResultsArrayPerWorkflowRun.push(
-                        (weightage.weightage *
-                          parseInt(chaosData.probeSuccessPercentage, 10)) /
-                          100
+                        chaosData.experimentVerdict === 'Fail'
+                          ? 0
+                          : (weightage.weightage *
+                              parseInt(chaosData.probeSuccessPercentage, 10)) /
+                              100
                       );
                       weightsSum += weightage.weightage;
                       isValid = true;
```

Comment:
Wanted to know what happens if the verdict is in the awaited state?

Reply:
@whitetiger1399 If it's in awaited we are getting 100 as the probe success percentage, which is causing the resiliency score to be non zero for even awaited experiment results which is incorrect. We have handled the case by explicitly using 0 even if the backend is not handling this, which currently is the case. cc: @ksatchit

Reply:
if the user schedules a workflow, then disables the workflow in the middle, so the workflow will remain in the awaited state and by current logic, it will act as if the workflow has passed. We should handle 3 cases for resiliency score calculation. If required, we can remove it from the calculation of the average resiliency socre

Reply:
Yes this change exactly handles that case @whitetiger1399

Reply:
Okay we can avoid it totally, adding that check too thanks @whitetiger1399